docs/content/docs/getting-started.md

```diff
@@ -73,14 +73,14 @@ No problem, you can still add a ``route`` field in the front-matter like this:
 ```md
 ---
 title: My super new page
-route: my-super-url.html
+route: my-super-url
```

Comment:
@MoOx, apologies for not being clear on this change.

At this point in the tutorial we are working with a file under `content/my-super-page.md`. The contents of that file are, at this point,
```md
---
title: My super new page
route: my-super-url.html
---

My super **content**!
```

Adding the custom route property with the `.html` extension in the front-matter is what is troubling. We are working within a markdown file, and I thought only markdown was supported by default with Phenomic.

With the `.html` extension, I am never able to get the page to load; I only ever see the 404 page. If I remove the extension, the content is available (at `http://localhost:3333/my-super-url/`); if I change the extension to be `.md`, the content is also available (at the same URL). Even changing the filetype to actually be an HTML file does not appear to work. Perhaps I am just not understanding what route the page should now be on?

Either way, it still feels jarring to me introducing the `.html` extension here, since we are never actually using or editing any HTML files in this tutorial.

I would love to hear your thoughts on this though. Again, I could just be overlooking or misunderstanding something about this functionality.

Thanks for taking the time to read this :)